src/index.js

```diff
@@ -1 +1,81 @@
 'use strict';
+
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
+const zlib = require('zlib');
+const { pipeline } = require('stream');
+
+const PORT = 5700;
+
+const server = http.Server();
+
+server.on('request', (req, res) => {
+  if(req.method === 'POST') {
+    const jsonPath = path.resolve('src', 'index.json');
+    const chunks = [];
+
+    if (!fs.existsSync(jsonPath)) {
+      res.readStatus = 500;
+      res.end()
+      return;
+    }
+    const jsonFile = fs.readFileSync(jsonPath, 'utf8');
+
+    if (!fs.existsSync(jsonFile)) {
+      res.readStatus = 500;
+      res.end()
+      return;
+    }
+    const jsonParse = JSON.parse(jsonFile);
+    
+    if (req.url.slice(1) === 'favicon.ico') {
+      return;
+    }
+
+    req.on('data', (chunk) => {    
+      chunks.push(chunk
+        .toString()
+        .split('&')
+        .map(info => info.split('=')));
+    });
```

Comment:
i think you just need to push chunk into chunks array, and then use something like
```
const data = Buffer.concat(chunks).toString();
const file = querystring.parse(data);
```

Reply:
I do not receive an object in the request, so it will not work, I checked before writing the code
i get data string date=&title=&amount=